sashimi/camera.py

```diff
@@ -153,6 +150,13 @@ def camera_loop(self):
             except Empty:
                 pass
 
+    def update_parameters(self):
+        self.parameters = self.new_parameters
+        self.camera.exposure_time = self.parameters
```

Comment:
this does not make sense: you should pass numbers/actual values to the setters, not a whole paramters object